composer.json

```diff
@@ -22,6 +22,7 @@
     "symfony/framework-bundle": "~2.8|~3.0",
     "symfony/browser-kit": "~2.8|~3.0",
     "symfony/expression-language": "~2.8|~3.0",
+    "symfony/console": "~2.8|~3.0",
```

Comment:
We have a problem with this dependency, an exception is thrown when it isn't loaded:
```
PHP Fatal error:  Class 'Symfony\Component\Console\Command\Command' not found in /home/guilhem/github/DunglasActionBundle/vendor/symfony/framework-bundle/Command/ContainerAwareCommand.php on line 23
```

What do you think about adding the default configuration values for the commands only when ``symfony/console`` is available (thanks to PrependExtensionInterface)